drools-ansible-rulebook-integration-protoextractor/src/test/java/org/drools/ansible/rulebook/integration/protoextractor/ExtractorTest.java

```diff
@@ -0,0 +1,100 @@
+package org.drools.ansible.rulebook.integration.protoextractor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+
+import org.drools.ansible.rulebook.integration.protoextractor.ast.ExtractorNode;
+import org.drools.model.Prototype;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ExtractorTest {
+
+    @Test
+    public void testBasic() throws Exception {
+        final String expression = "a.b[\"c\"].d[\"e/asd\"]['f'].g.h";
```

Comment:
Do you know if we have a spec/grammar/TCK to make sure that this is covering all the options supported by jinja?

Reply:
since `ansible-rulebook` at this point specify its own parser (not inherit) from jinjia we have no other choice but to keep manually alinged.

This point was already raised to ansible-rulebook team! 👍